dir.targets

```diff
@@ -95,7 +95,7 @@
 
     <!-- Download latest nuget.exe -->
     <DownloadFile FileName="$(NuGetToolPath)"
-                  Address="https://dist.nuget.org/win-x86-commandline/v3.2.0-rc/nuget.exe"
+                  Address="https://dist.nuget.org/win-x86-commandline/v3.2.0/nuget.exe"
```

Comment:
Ok, I'll go with the explicit 3.2 for now and we'll see if the churn gets too annoying.